pkg/pss/pss.go

```diff
@@ -85,17 +89,18 @@ func (p *Pss) Register(topic trojan.Topic, hndlr Handler) {
 }
 
 // Deliver allows unwrapping a chunk as a trojan message and calling its handler func based on its topic
-func (p *Pss) Deliver(c swarm.Chunk) {
+func (p *Pss) Deliver(c swarm.Chunk) error{
 	if trojan.IsPotential(c) {
+		ctx := context.Background()
```

Comment:
Would it be better to accept the context in Deliver function? This context here is not so useful for cancelation.